### ISSUE TITLE
In Query Schema, `__typename` field always have type `"__typename"`

### ISSUE THREAD

**Issue by tomocrafter** [uhyo/nitrogql#46]
### What happened?
It appears that adding `__typename` field to query results in incorrect `"__typename"` type.

### What did you expect?
I expect to this query:
```graphql
query User {
  user(id: 1) {
    __typename
  }
}
```
to be generated type as:
```typescript
export type UserFragment = Schema.__SelectionSet<Schema.__OperationOutput.User, {
  __typename: "User";
}
```
instead `"__typename"` string type:
```typescript
export type UserFragment = Schema.__SelectionSet<Schema.__OperationOutput.User, {
  __typename: "__typename";
}
```

The type in `Schema.__OperationOutput.User` has correct type, `__typename: "User"`. but in `SelectionSet`, It always have `"__typename"` which is incorrect.

I suspect here: https://github.com/uhyo/nitrogql/blob/aa10bf177b803df2a2f1cc3d8a9262564540ff7b/crates/printer/src/operation_type_printer/selection_tree/to_ts.rs#L93

**Comment by uhyo** [comment]
What a shame... I believe this is a recent regression. Working on a fix :bow:

**Comment by uhyo** [comment]
Sorry for this, should be fixed in `1.6.3` 🥲

**Comment by tomocrafter** [comment]
tried it, and now it works as expected perfectly! thank you for quick fix!
